gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcWriteChannel.java

```diff
@@ -393,7 +392,7 @@ public void onError(Throwable t) {
                   String.format(
                       "Caught exception for '%s', while uploading to uploadId %s at writeOffset %d."
                           + " Status: %s",
-                      resourceId, uploadId, writeOffset, statusDesc),
```

Comment:
This assumes `s` to be always non-null, is this guaranteed tho?

Reply:
Yes I believe Status.fromThrowable won't return null. And we may want the system to throw NullPointerException if that happens as it's unexpected. WDYT?

Reply:
I also think it wouldnt. Let’s keep it this way for now.